docs/brillig/00_intro.md

```diff
@@ -0,0 +1,47 @@
+# Intro to Brillig
+BRILLIG - Bytecode Runtime Interface for Liberated Language Implementation... G
+
+The G stands for whatever you like, like [Gabagool](https://en.wikipedia.org/wiki/Gabagool).
+
+Noir can compile to bytecode for Brillig, a general VM architecture for use with Noir.
```

Comment:
I feel like we can mention Noir as an example of who can use Brillig, but we want Brillig to live on its own. Stating that Brillig is a general VM architecture for use with Noir first doesn't seem right. What if we switched this?

Something like:
"Brillig is a general VM architecture for usage inside of a circuit language that requires unconstrained or non-deterministic functionality. For example, a language which compiles down to ACIR, can integrate unconstrained functions into its circuits by also compiling down to Brillig bytecode."